website/events/helpers/models.py

```diff
@@ -40,7 +45,7 @@ class SaveReversionMixin:
     '''Mixin to override save, wrapping with create_revision.
     To better work keep on left on inheritance.'''
     def save(self, *args, **kwargs):
-        with reversion.create_revision():
+        with reverse.create_revision():
```

Comment:
Why this change?

Reply:
again reversion was not working so I changed it.
Hope it explains all questions , thanks for checking out my PR.

Reply:
reversion (https://django-reversion.readthedocs.io/en/stable/index.html) is now part of django.urls?.

Reply:
But `reverse` does not have a `create_revision` function inside...

Reply:
Oh.. , I mistakenly did not checked it . So now , should I make a funtion of  create_revision or keep it the same as previous?

Reply:
I still don't know why you want to change this.